templates/api.hbs

```diff
@@ -1,17 +1,24 @@
 {{>header}}
 
-import { Express } from 'express'
+import { Express, Request, Response } from 'express'
 import passport from 'passport'
 import * as t from './types'
 import * as v from '../../validation'
+{{#if (isAnyOperationSupportMultipart operations)}}
+import * as f from '../../impl/helpers/{{lowerCase name}}MultipartHelper'
+{{/if}}
 import { Api } from '../../models'
 
 export default function(app: Express, impl: t.{{className name}}Api) {
 	{{#each operations}}
 	app.{{lowerCase httpMethod}}(
 		{{{stringLiteral (pathTemplate fullPath)}}},
+		{{#if (isOperationSupportsMultipart .)}}
+		f.upload.fields([{{#with (filterMultipartFiles requestBody.defaultContent.schema.properties)}}{{#each this}}{name: {{{stringLiteral name}}}, maxCount: 1}{{#hasMore}}, {{/hasMore}}{{/each}}{{/with}}]),
```

Comment:
We could perhaps change `filterMultipartFiles` to be `multipartFileProperties` and simply give it the `requestBody` or `requestBody.defaultContent` rather than needing to go into `schema.properties`...

Reply:
Maybe rather than `name` we should use `serializedName` here... `name` will have been through the identifier transformation, whereas `serializedName` is the name that is used on the wire.

Reply:
This loop could be `{{#each (multipartFileProperties requestBody.defaultContent)}}` rather than the `with` and then the `each`?